### ISSUE TITLE
"missing whitespace" error when using the 'required' option for a secret mount in RUN

### ISSUE THREAD

**Issue by Chili-Man** [hadolint/hadolint#444]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
Hadolint should recognize that the "required"  option is part of the parameters for 
```Dockerfile
RUN --mount=type=secret
```

as specified in the documentation: https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/experimental.md#run---mounttypesecret

### Actual behavior

Hadolint prints the error message "missing whitespace"

### Steps to reproduce the behavior

Try linting the following Dockerfile:

```dockerfile
FROM alpine

RUN --mount=type=secret,id=secret,required=true ls -la
```

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter 1.18.0-no-git
```

**Dockerfile** (if relevant)

```Dockerfile
FROM alpine

RUN --mount=type=secret,id=secret,required=true ls -la
```

**Additional environment details (OS, stack version, etc.)**
I'm running on Ubuntu 20.04

**Comment by lorenzo** [comment]
You just need to write required, not required=true according to the docs you linked. Or am I misreading it?

**Comment by Chili-Man** [comment]
hmm, well I tried it both ways with `required=true` and `required` and both seem to acceptable by the docker build. I suppose that can be a workaround for now, but the way I read the doc was that it should be in the form of `required=true` and that is consistent with the way the rest of the arguments are set.

**Comment by lorenzo** [comment]
unfortunately, the docs are quite misleading for this experimental feature. I had never seen a `required=true` before, so that took me by surprise :)
